package.json

```diff
@@ -41,8 +41,7 @@
     "react-dom": "^16.8.6",
     "react-modal": "^3.8.2",
     "styled-components": "^4.3.2",
-    "supertest": "^4.0.2",
-    "webpack": "^4.35.3"
+    "supertest": "^4.0.2"
```

Comment:
Not sure what supertest is used for, but looks good to me

Reply:
Looks good.  We used supertest to test api end points.